src/TwigComponent/tests/Fixtures/templates/components/NotAComponent.md

```diff
@@ -0,0 +1 @@
+This file should not cause issues with the debug:twig:component command.
```

Comment:
Maybe add a case in TwigComponentDebugTest that check NotAComponent is not listed ?

(like the `testWithNoMatchComponent` test ?)